eras/shelley-ma/impl/src/Cardano/Ledger/Mary/Value.hs

```diff
@@ -509,8 +540,8 @@ to ::
   -- The Nothing case of the return value corresponds to a quantity that is outside
   -- the bounds of a Word64. x < 0 or x > (2^64 - 1)
   Maybe (CompactValue crypto)
-to (MaryValue ada ma)
-  | Map.null ma =
+to (MaryValue ada m)
+  | m == mempty =
       CompactValueAdaOnly . CompactCoin <$> integerToWord64 ada
```

Comment:
Like this maybe

to (MaryValue ada (MultiAsset m))
  | Map.null m =
      CompactValueAdaOnly . CompactCoin <$> integerToWord64 ada